r/Dockerfile.conda

```diff
@@ -26,9 +26,30 @@ RUN conda install -q \
         nomkl && \
     conda clean --all
 
-ENV ARROW_PYTHON=OFF \
-    ARROW_BUILD_TESTS=OFF \
-    R_CONDA=1
+# Ensure parallel R package installation and set CRAN repo mirror
+RUN printf "options(Ncpus = parallel::detectCores(), repos = 'https://demo.rstudiopm.com/all/__linux__/bionic/latest')\n" >> /opt/conda/lib/R/etc/Rprofile.site
```

Comment:
I think conda installs R package dependencies separately/differently, so this "repos" arg probably doesn't do anything (well, it definitely doesn't do anything without the `HTTPUserAgent` suggestion I made above, but I don't know that it would matter even if you added it).